tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -897,7 +897,6 @@ public function callAPISuccessGetValue($entity, $params, $type = NULL) {
   public function callAPISuccessGetSingle($entity, $params, $checkAgainst = NULL) {
     $params += array(
       'version' => $this->_apiversion,
-      'debug' => 1,
```

Comment:
Hmm - it also improves test information available I think - I know the getAndCheck function handles this parameter